packages/cryptoplease/lib/features/favorite_tokens/src/favorite_tokens_list.dart

```diff
@@ -145,7 +137,11 @@ class _TokenItem extends StatelessWidget {
         overflow: TextOverflow.ellipsis,
       ),
       trailing: Text(
-        tokenRate?.format(locale) ?? '-',
+        _formatPrice(
+          currentPrice?.toDouble(),
+          symbol: fiatCurrency.sign,
+          maxDecimals: 5,
```

Comment:
Why are there different max decimals for this list and for details screen?

And why is this logic for formatting repeated?

Reply:
I've updated code to remove repeating logic and the same decimals.

Initially set them differently since list has smaller space compared to details, but upon rechecking it can be set the same.